agate/pattern-single-panel/src/components/ProfilePhotoPicker.module.less

```
@@ -1,3 +1,7 @@
+.bodytext {
+	margin: 0px;
```

```suggestion
	margin: 0;
```

https://www.sitepoint.com/community/t/whats-the-best-0px-or-0/5306
There is no reason to add this unit.

---

I understand. It's fixed.